torch/_prims/__init__.py

```diff
@@ -259,22 +238,13 @@ def TensorMeta(
     if isinstance(device, str):
         device = torch.device(device)
 
-    if isinstance(tensorlike, FakeTensor):
-        mode = tensorlike.fake_mode
-    else:
-        mode = get_prim_fake_mode()
-
-    if device.type == "meta":
-        return torch.empty_strided(shape, strides, dtype=dtype, device="meta")
+    # SymInt doesnt support empty_strided yet
+    if any(
+        isinstance(inp, torch.SymIntNode) for inp in itertools.chain(shape, strides)
+    ):
+        return torch.empty(shape, dtype=dtype, device=device)
```

Comment:
hmm, this is failing now..

Reply:
when I invoke this with `torch.empty([2, 3], dtype=dtype, device=device)` here a FakeTensor is returned successfully but `torch.ops.aten.empty.SymInt` with SymInt inputs fails with `RuntimeError: SymIntArrayRef expected to contain only concrete integers

cc @horace, @Krovatkin